bin/appcenter.js

```diff
@@ -58,6 +54,11 @@ function runCli() {
   });
 }
 
+async function showUpdateNotification(pkg) {
+  const { default: updateNotifier } = await import("update-notifier");
```

Comment:
What the reason to change `require` to `import`

Reply:
Because update-notifier switched to pure ESM. See https://github.com/yeoman/update-notifier/releases/tag/v6.0.0.
This means we can't use normal `require` anymore.